docker/conf/docker-compose-mac.yml.dist

```diff
@@ -1,7 +1,7 @@
 version: "3.1"
 services:
     postgres:
-        image: postgres:9.5-alpine
+        image: postgres:10-alpine
```

Comment:
I remember that we have created a rule for using as many concrete versions of Docker images as possible. In parameters.yml.dist you say that version 10.4 is used. I think that 10.4-alpine image should be used here.